### ISSUE TITLE
49 fails out of 49

### ISSUE THREAD

**Issue by garex** [mrkmg/git-stream#4]
What am I missing?

```
...
49 tests, 49 failures
Makefile:31: recipe for target 'test' failed
make: *** [test] Error 1

git --version
git version 2.17.1

 cat /etc/issue
Ubuntu 18.04.1 LTS \n \l

bash --version
GNU bash, версия 4.4.19(1)-release (x86_64-pc-linux-gnu)
```

**Comment by mrkmg** [comment]
Hello garax,

Could you paste in the method you are running the tests? Did you clone the git submodules?

Kevin

**Comment by garex** [comment]
@mrkmg yes. I just follow https://github.com/mrkmg/git-stream#installation

**Comment by mrkmg** [comment]
I have tested on a fresh install of ubuntu with git and make installed and everything seemed to work. Not sure how to help here.

**Comment by garex** [comment]
Will be fixed in pull request #5

**Comment by mrkmg** [comment]
Thank you!
